Tools/create_cert.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+openssl req -subj '/CN=XcodeSigner' -config ./Tools/cert.config -x509 -newkey rsa:2046 -keyout selfSignedKey.pem -out selfSigned.pem -days 365 -passout pass:"foobar"
+openssl pkcs12 -export -out XcodeSigner.p12 -inkey selfSignedKey.pem -in selfSigned.pem -passin pass:"foobar" -passout pass:"foobar"
+security import XcodeSigner.p12 -P foobar
```

Comment:
this script create self-signed cert on CLI but that need too long time.
finally, I choose unsign way for unit testing.

WARN: don't use unsign way in usual use case because stuck some other application with `tccd` process burn.